packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -119,7 +121,19 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
               <Icon iconName={ getRTL() ? 'ChevronLeft' : 'ChevronRight' } />
             </span >
           </div >
-          <div className={ css('ms-DatePicker-headerToggleView js-showMonthPicker', styles.headerToggleView) } />
+          {
+            showMonthPickerAsOverlay ?
+              <div
+                className={ css('ms-DatePicker-headerToggleView js-showMonthPicker', styles.headerToggleView) }
+                onClick={ () => this._onSelectSwitchCalendar(false) }
+                onKeyDown={ this._onKeyDown.bind(this, () => this._onSelectSwitchCalendar(true)) }
+                aria-label={ strings.monthPickerAriaLabel }
+                role='button'
+                tabIndex={ 0 }
```

Comment:
Someone created this div ahead of me to be used to toggle the header. The css had this message:
// Without modifying the Pickadate JS, this transparent
// button is necessary to toggle the month view.
.headerToggleView { ...... }

I'm not sure what that means, but that is why I used it.